src/nunavut/jinja/lang/__init__.py

```diff
@@ -32,7 +32,7 @@ def get_supported_languages() -> typing.Iterable[str]:
     return __language_modules__.keys()
 
 
-def add_language_support(language_name: str, environment: Environment) -> None:
+def add_language_support(language_name: str, environment: Environment, make_implicit: bool = False) -> None:
```

Comment:
Missing docs for `make_implicit`

Reply:
Oh, good catch. I also forgot to test for a double call to this.